docs/docsite/rst/shared_snippets/basic_concepts.txt

```diff
@@ -1,34 +1,56 @@
 Control node
 ============
 
-Any machine with Ansible installed. You can run Ansible commands and playbooks by invoking the ``ansible`` or ``ansible-playbook`` command from any control node. You can use any computer that has a Python installation as a control node - laptops, shared desktops, and servers can all run Ansible. However, you cannot use a Windows machine as a control node. You can have multiple control nodes.
+The machine from which you run the Ansible CLI tools (with Ansible installed), the ``ansible`` or ``ansible-playbook`` binaries. You can use any computer that meets the software requirements as a control node - laptops, shared desktops, and servers can all run Ansible. You can have multiple control nodes, but Ansible itself does not coordinate across them, see AAP for a .
 
 Managed nodes
 =============
 
-The network devices (and/or servers) you manage with Ansible. Managed nodes are also sometimes called "hosts". Ansible is not installed on managed nodes.
+The target devices (servers, network appliances or any computer) you manage with Ansible. Managed nodes are also sometimes called "hosts". Ansible is not installed on managed nodes.
 
 Inventory
 =========
 
-A list of managed nodes. An inventory file is also sometimes called a "hostfile". Your inventory can specify information like IP address for each managed node. An inventory can also organize managed nodes, creating and nesting groups for easier scaling. To learn more about inventory, see :ref:`the Working with Inventory<intro_inventory>` section.
+A list of managed nodesi provided by one or more 'inventory sources'. Your inventory can specify information like IP address for each managed node,.organize managed nodes into groups and assigning data specific to the group or host by using variables. To learn more about inventory, see :ref:`the Working with Inventory<intro_inventory>` section. Sometimes an inventory source file is also reffered to as a 'hostfile'.
 
-Collections
-===========
+Playbooks
+=========
+
+They contain Plays (which are the basic unit of Ansible exeuction), this is both an 'execution concept' and how we describe the files on which ``ansible-playbook`` operates on.
```

Comment:
```suggestion
They contain Plays (the basic unit of Ansible execution), this is both an 'execution concept' and how we describe the files on which ``ansible-playbook`` operates.
		Playbooks are written in YAML and are easy to read, write, share and understand. To learn more about playbooks, see :ref:`about_playbooks`.
```

also where it goes "this is both an `execution concept`", what is both an execution concept ?